File: Starter Problems 3/ency.py
op = open('encyin.txt', 'r')
n, q = op.readline().split()
n = int(n)
q = int(q)
dic = {}

for i in range(1, n + 1):
	dic[str(i)]=(op.readline().strip())


ot = open('encyout.txt', 'w')
for i in range(0, q):
	ot.write(dic[op.readline().strip()])
	ot.write('\n')

File: Starter Problems 3/ency.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ifstream inFile;
    inFile.open("encyin.txt");
    int n,q;
    inFile >> n;
    inFile >> q;
    int nums[n];
    for (int i=0;i<n;i++) {
        inFile >> nums[i];
    }
    ofstream outFile;
    outFile.open("encyout.txt");
    int temp;
    for (int i=0;i<q;i++) {
        inFile >> temp;
        outFile << nums[temp-1] << "\n";
    }
    inFile.close();
    outFile.close();
}
